tinyengine/core.py

```diff
@@ -1,7 +1,7 @@
 import pygame
-from pygame.locals import *
```

Comment:
There are some features that is imported by locals. Did you checked if the code is not using any of those?

Reply:
yep

Reply:
locals are no longer being used